src/main/java/com/facebook/openwifirrm/ucentral/models/AggregatedState.java

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifirrm.ucentral.models;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import com.facebook.openwifirrm.ucentral.models.State.Interface.SSID.Association;
+import com.facebook.openwifirrm.ucentral.models.State.Interface.SSID.Association.Rate;
+
+/**
+ * Aggregation model for State aggregation. Only contains info useful for analysis.
+ */
+public class AggregatedState {
```

Comment:
Please rearrange classes, members and methods in this classes to align with the styles with others.